### ISSUE TITLE
Show

### ISSUE THREAD

**Issue by davidagold** [davidagold/_AbstractTables.jl#1]
Currently, `show`ing an object of type `T <: AbstractTable` that satisfies the column-indexable interface will cut off columns at the edge of the display, as opposed to `show`ing in chunks. This is my desired default behavior for `show`, but I implemented it
1. at the expense of being able to do `showall` in order to see all the columns shown in chunks
2. without providing any information about the columns _not_ shown, as is done in `tibble`s. 

Both of those shortcomings ought to be corrected.

**Comment by nalimilan** [comment]
See also https://github.com/JuliaStats/DataFrames.jl/pull/995.

**Comment by davidagold** [comment]
This situation is improved in https://github.com/davidagold/_AbstractTables.jl/commit/60e65a176bca71a0adeefd25afc9e16c923ec038

In particular, one can now show all columns in chunks or not, and if columns are cut off from the display, then the footer of the table will include information about the columns that are not shown.

EDIT: I'm going to leave this issue open, however, as a forum for discussion of show methods for tables. There is still work to be done. A further improvement I'd like to make is to include type information for each column.

**Comment by nalimilan** [comment]
See also https://github.com/tidyverse/tibble/blob/master/README.md for ideas about how to print tables.

**Comment by davidagold** [comment]
Yes, Hadley's `tibble` printing has been an inspiration :) The main differences between that and the current `show` regime in this package is that `tibble`s do not print separators (e.g.`|`) and `tibble`s do print the column eltypes. I think I prefer both of those behaviors, and would be willing to incorporate them here.
